index.html

```diff
@@ -784,10 +784,18 @@ <h3>Proofs</h3>
 
           <dt>verificationMethod</dt>
           <dd>
-The means and information needed to verify the proof MUST be specified as a
-string that maps to a [[URL]]. An example of a verification method is a
-link to a [=public key=] which includes cryptographic material that is used
-by a verifier during the verification process.
+If included, `verificationMethod`, the means and information needed to verify the proof,
+MUST be specified as a string that maps to a [[URL]]. An example of a verification method
+is a link to a [=public key=] which includes cryptographic material that is used
+by a verifier during the verification process. Inclusion of `verificationMethod` is OPTIONAL, 
+but if it is not included, other properties such as `cryptosuite` MUST provide a mechanism
+by which to obtain the required information to verify the proof.
```

Comment:
This "MUST" statement is not testable. How would we write a test to verify that a system is conformant to this mandatory requirement? This seems to be a normative statement on a specification requirement, which we should avoid unless it's absolutely necessary.

Reply:
Perhaps this needs to say that a cryptosuite specification is required to provide the details for how to obtain the verification material, not using MUST language, but still making it a requirement on a cryptosuite spec (and not a detail that could be tested against an implementation).

Reply:
Agree with @dlongley. The change proposed by @TallTed below seems to cover the type of normative requirement we have, so a MUST language here seems to be superfluous...

Reply:
Fixed in 15ff00b19f64b27ccf42cb6d20a1a2f6189b638a.